x-pack/solutions/security/plugins/security_solution/public/detections/components/alerts_table/timeline_actions/alert_context_menu.tsx

```diff
@@ -300,7 +303,7 @@ const AlertContextMenuComponent: React.FC<AlertContextMenuProps> = ({
 
   return (
     <>
-      {items.length > 0 && (
+      {items.length > 0 ? (
```

Comment:
Just curious on why keeping this check here? I tried just completely removing it and it seemed to work just fine, as the `disabled` prop on the `EuiButtonIcon` takes care of everything.

Reply:
Good point!

I was afraid the `EuiPopover` would interfere with the tooltip but after some testing it doesn't 😄